### DISCUSSION TITLE
Vercel with AWS EC2 Elastic IP

### DISCUSSION THREAD

**Discussion by syongkheng** [vercel/community#6858] [Help]
### Summary

Hello everyone,

I have a domain with Vercel i.e. domain.dev which serves my frontend. The project settings is linked to the frontend repository.

I have a separate backend server which is in NodeJS and the instance is running in AWS EC2. I am able to access the address and receive the following response: 
![image](https://github.com/vercel/community/assets/33014950/9af1fbdd-5f6e-438b-abf5-531a5f272968)

I want to link the NodeJS server as a subdomain i.e. api.domain.dev, and have done the following:
![image](https://github.com/vercel/community/assets/33014950/41a2ad4d-733f-421f-8532-d858aadcba4f)
(added A record with the IP pointing to the Elastic IP of the EC2 Instance).

However, when I tried to navigate to the page i.e. api.domain.dev, I encountered "The site can't be reached, api.domain.dev refused to connect". Removing the sub domain i.e. domain.dev shows that the frontend is still up and running.

I am unsure of how I can proceed from here, do I just have to wait it out for the DNS to be configured?

Thanks in advanced and any advice would be greatly appreciated.

### Steps to Reproduce

1) Create EC2 Instance in AWS and assign elastic ip
2) Server is accessible through the IP 
3) Added A record under DNS of my Vercel project
4) Accessing the subdomain results in the page refusing to connect

**Comment by syongkheng** [comment]
Hi, 

I have managed to get it to work. Will proceed to close the discussion.

**Reply by arian0zen** [reply]
hey, I am stuck in this exact same situation
i would be very kind of you to share how did you proceed from this situation!
thank you

**Reply by altezza04** [reply]
Me too, please provide step how you manage to fix it

[a comment by an automated account is not shown]
